### PR TITLE
Do not suggest User in intellisense

### PR DESCRIPTION
**Pull request by jas-valgotar**
Fixes https://github.com/microsoft/Power-Fx/issues/1278

### CODE REVIEW

**Comment by LucGenetier** [comment]
✅ No public API change.

**Comment by LucGenetier** [comment]
✅ No public API change.

File: src/libraries/Microsoft.PowerFx.Core/Texl/Intellisense/IntellisenseData/IntellisenseData.cs [hunk 1]
```diff
@@ -312,7 +314,10 @@ internal virtual void AddCustomSuggestionsForValuePossibilities()
         /// </summary>
         internal virtual void AddCustomSuggestionsForGlobals()
         {
-            foreach (var global in _powerFxConfig.GetSuggestableSymbolName())
+            // Do not suggest the user info symbol.
+            var symbols = _powerFxConfig.GetSuggestableSymbolName().Where(symbol => symbol != SymbolTable.UserInfoSymbolName);
```
Comment (MikeStall) [inline src/libraries/Microsoft.PowerFx.Core/Texl/Intellisense/IntellisenseData/IntellisenseData.cs:318]: >symbol != SymbolTable.UserInfoSymbolName [](http://example.com/codeflow?start=84&length=40)

?

File: src/tests/Microsoft.PowerFx.Core.Tests/IntellisenseTests/SuggestTest.cs [hunk 2]
```diff
@@ -473,6 +473,21 @@ public void SuggestBlockImplicitThisRecord(string expression, params string[] ex
...
+        [Theory]
+        [InlineData("Use|", "Color.Chartreuse", "User")]
+
+        //Should Not Suggest User here.
+        [InlineData("Power(|")]
+        public void SuggestUser(string expression, params string[] expected)
+        {
+            var config = SuggestTests.Default;
+
+            config.SymbolTable.AddUserInfoObject();
```
Comment (MikeStall) [inline src/tests/Microsoft.PowerFx.Core.Tests/IntellisenseTests/SuggestTest.cs:485]: >config.SymbolTable. [](http://example.com/codeflow?start=12&length=19)

Related test:
config.SymbolTable.AddConstant("Foo", 123) ?

**Review by MikeStall — APPROVED** [review]
:shipit:

File: src/libraries/Microsoft.PowerFx.Core/Texl/Intellisense/IntellisenseData/IntellisenseData.cs [hunk 3]
```diff
@@ -312,16 +314,6 @@ internal virtual void AddCustomSuggestionsForValuePossibilities()
...
-            foreach (var global in _powerFxConfig.GetSuggestableSymbolName())
-            {
-                DType type = default;
-                if (_powerFxConfig.GetSymbols(global, out var nameInfo))
-                {
-                    type = nameInfo.Type;
-                }
-
-                IntellisenseHelper.AddSuggestion(this, global, SuggestionKind.Global, SuggestionIconKind.Other, type, requiresSuggestionEscaping: true);
-            }
```
Comment (jas-valgotar) [inline src/libraries/Microsoft.PowerFx.Core/Texl/Intellisense/IntellisenseData/IntellisenseData.cs:324]: This is overridden in PA, so no impact to PA.

**Review by MikeStall — APPROVED** [review]

**Comment by LucGenetier** [comment]
✅ No public API change.

**Pull request merged**